Mathlib/Data/Int/WithZero.lean

```diff
@@ -88,4 +88,38 @@ theorem toNNReal_strictMono {e : NNReal} (he : 1 < e) :
       WithZero.coe_unzero hy]
     exact hxy
 
+theorem toNNReal_eq_one_iff {e : NNReal} (m : ℤₘ₀) (he0 : e ≠ 0) (he1 : e ≠ 1) :
```

Comment:
Would you mind replacing all of the `{e : NNReal} ` in the file by `{e : ℝ≥0}`? (I know that not all of these were introduced by this PR, so I will leave it up to you).

Reply:
Done!